### ISSUE TITLE
Database performance isn't measured anymore

### ISSUE THREAD

**Issue by annando** [friendica/friendica#7426]
### Expected behavior
With the activated "rendertime" addon, the database performance should be measured. Also when the "profiler" config value is set, individual database performance data should be collected.

### Actual behavior
No data is collected for database actions:
```
Database: 0/0, Network: 0, Rendering: 0.04, Session: 0, I/O: 0.01, Other: 1.21, Total: 1.26

Database Read:

Database Write:

Cache Read:

Cache Write:

Network:
```

### Steps to reproduce the problem
Activate the "rendertime" addon, activate the "profiler" config value.

### Friendica version you encountered the problem
Current develop branch.

**Comment by nupplaphil** [comment]
I think I know what happened ;) I'll fix it

**Comment by nupplaphil** [comment]
Should be fixed by #7427 
(Tested on local instance with everything printed as expected :-) )

**Comment by annando** [comment]
It's working fine again:
```
Database: 2.868/0.045, Network: 0, Rendering: 0.11, Session: 0, I/O: 0.02, Other: 0.47, Total: 3.51

Database Read:
Database::select, Database::selectFirst, Config::get, JitConfiguration::get: 0.044
Database::select, DBA::select, Hook::loadHooks, App::reload: 0.001
DBA::fetch, Hook::loadHooks, App::reload, App::__construct: 0.002
Database::selectFirst, Config::get, JitConfiguration::get, Config::get: 0.002
Database::select, Database::selectFirst, PConfig::get, JitPConfiguration::get: 0.008
Database::selectFirst, PConfig::get, JitPConfiguration::get, PConfig::get: 0.002
Database::select, Database::selectFirst, DBA::selectFirst, Session::setAuthenticatedForUser: 0.001
DBA::fetch, Hook::loadHooks, App::runFrontend: 0.001
Database::fetchFirst, DBA::fetchFirst, Profile::getByNickname, Profile::load: 0.001
q, Profile::sidebar, Profile::load, Profile::content: 0.001
Database::fetchFirst, Database::count, DBA::count, ContactBlock::getHTML: 0.018
Database::select, DBA::select, ContactBlock::getHTML, Profile::sidebar: 0.002
DBA::fetch, ContactBlock::getHTML, Profile::sidebar, Profile::load: 0.001
Database::select, Database::selectFirst, DBA::selectFirst, Contact::getProbeDataFromDatabase: 0.103
Database::selectFirst, DBA::selectFirst, Contact::getProbeDataFromDatabase, Contact::magicLink: 0.008
Database::fetchFirst, Database::count, DBA::count, Profile::sidebar: 0.017
DBA::p, Profile::content, call_user_func, App::runFrontend: 0.042
DBA::p, Profile::getBirthdays, Profile::content, call_user_func: 0.005
Database::select, Database::selectFirst, DBA::selectFirst, Contact::getIdForURL: 0.062
Database::select, Database::exists, DBA::exists, Worker::add: 0.051
Database::exists, DBA::exists, Worker::add, Contact::getIdForURL: 0.002
Database::e, Database::insert, DBA::insert, Worker::add: 0.017
Database::fetchFirst, Database::count, DBA::count, Worker::activeWorkers: 0.029
DBA::p, Worker::tooMuchWorkers, Worker::add, Contact::getIdForURL: 1.262
DBA::fetch, Worker::tooMuchWorkers, Worker::add, Contact::getIdForURL: 0.011
DBA::close, Worker::tooMuchWorkers, Worker::add, Contact::getIdForURL: 0.007
Database::fetchFirst, Database::count, DBA::count, Worker::deferredEntries: 0.326
Database::select, Database::exists, DBA::exists, Worker::entriesExists: 0.037
Database::exists, DBA::exists, Worker::entriesExists, Worker::tooMuchWorkers: 0.002
Database::select, Database::selectFirst, Database::update, DBA::update: 0.017
Database::selectFirst, Database::update, DBA::update, Worker::IPCSetJobState: 0.002
Database::e, Database::update, DBA::update, Worker::IPCSetJobState: 0.006
DBA::p, Item::select, Item::exists, Profile::getEventsReminderHTML: 0.002
DBA::p, Item::select, Item::exists, Profile::content: 0.001
Database::select, DBA::select, Item::update, Profile::content: 0.001
Database::e, Database::update, DBA::update, Item::update: 0.002
DBA::p, Item::select, Item::selectFirst, Item::updateThread: 0.024
DBA::fetch, Item::fetch, Item::selectFirst, Item::updateThread: 0.001
DBA::close, Item::selectFirst, Item::updateThread, Item::update: 0.001
DBA::p, Item::select, Item::selectForUser, conversation_add_children: 0.083
DBA::fetch, Item::fetch, conversation_fetch_comments, conversation_add_children: 0.012
Database::select, DBA::select, Term::tagArrayFromItemId, Term::fileTextFromItemId: 0.041
DBA::close, conversation_fetch_comments, conversation_add_children, conversation: 0.001
DBA::p, Item::selectThread, Item::selectFirstThread, Item::selectFirstThreadForUser: 0.021
DBA::fetch, Item::fetch, Item::selectFirstThread, Item::selectFirstThreadForUser: 0.001
DBA::close, Item::selectFirstThread, Item::selectFirstThreadForUser, Post::getTemplateData: 0.001
Database::fetchFirst, DBA::fetchFirst, User::getOwnerDataById, Post::getDefaultText: 0.11
Database::select, Database::selectFirst, DBA::selectFirst, Photo::selectFirst: 0.058
Database::selectFirst, DBA::selectFirst, Photo::selectFirst, User::getOwnerDataById: 0.004
DBA::p, Item::select, Item::selectFirst, Post::getDefaultText: 0.034
DBA::fetch, Item::fetch, Item::selectFirst, Post::getDefaultText: 0.002
DBA::close, Item::selectFirst, Post::getDefaultText, Post::getCommentBox: 0.002
Database::select, DBA::select, Term::tagArrayFromItemId, Post::getDefaultText: 0.043
Database::select, DBA::select, Term::populateTagsFromItem, Item::prepareBody: 0.057
DBA::fetch, Term::populateTagsFromItem, Item::prepareBody, Post::getTemplateData: 0.003
DBA::close, Term::populateTagsFromItem, Item::prepareBody, Post::getTemplateData: 0.017
Database::select, DBA::select, Term::populateTagsFromItem, Post::getTemplateData: 0.105
DBA::fetch, Term::populateTagsFromItem, Post::getTemplateData, Thread::getTemplateData: 0.001
Database::selectFirst, DBA::selectFirst, Contact::getIdForURL, item_photo_menu: 0.004
Database::select, Database::selectFirst, DBA::selectFirst, item_photo_menu: 0.045
Database::selectFirst, DBA::selectFirst, item_photo_menu, Post::getTemplateData: 0.004
Database::select, Database::selectFirst, DBA::selectFirst, ContactSelector::networkToName: 0.069
Database::selectFirst, DBA::selectFirst, ContactSelector::networkToName, Post::getTemplateData: 0.007
DBA::p, Item::select, Item::selectFirst, Post::getTemplateData: 0.011
DBA::fetch, Item::fetch, Item::selectFirst, Post::getTemplateData: 0.001
DBA::close, Item::selectFirst, Post::getTemplateData, Post::getTemplateData: 0.001
Database::toArray, DBA::toArray, Term::tagArrayFromItemId, Post::getDefaultText: 0.003
DBA::p, Contact::getDetailsByURL, Post::getDefaultText, Post::getCommentBox: 0.012
Database::toArray, DBA::toArray, Contact::getDetailsByURL, Post::getDefaultText: 0.001
Database::select, DBA::select, Item::update, Item::putInCache: 0.011
DBA::fetch, Item::update, Item::putInCache, Item::prepareBody: 0.001
Database::selectFirst, Database::update, DBA::update, Item::updateContent: 0.002
Database::e, Database::update, DBA::update, Item::updateContent: 0.016
DBA::close, Item::update, Item::putInCache, Item::prepareBody: 0.001
DBA::fetch, Term::populateTagsFromItem, Post::getTemplateData, Post::getTemplateData: 0.002
DBA::close, Term::populateTagsFromItem, Post::getTemplateData, Post::getTemplateData: 0.001
Database::e, Database::update, DBA::update, Item::updateThread: 0.003

Database Write:
Database::insert, DBA::insert, Worker::add, Contact::getIdForURL: 0.017
Database::update, DBA::update, Worker::IPCSetJobState, Worker::tooMuchWorkers: 0.006
Database::update, DBA::update, Item::update, Profile::content: 0.002
Database::update, DBA::update, Item::updateContent, Item::update: 0.016
Database::update, DBA::update, Item::updateThread, Item::update: 0.003

Cache Read:
Profile::getBirthdays, Profile::content, call_user_func, App::runFrontend: 0.001

Cache Write:

Network:

```
